starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -81,6 +75,16 @@ async def get_events(
     This is why we need to iterate block by block, take all events,
     and chunk it later which is not an optimal solution.
     """
+    # Required parameters
+    from_block = filter.get("from_block")
+    to_block = filter.get("to_block")
+    chunk_size = int(filter.get("chunk_size"))
```

Comment:
what happens if `chunk_size` is not convertable to `int`? Does it lead to a 500 error?

Reply:
Yes it will lead to 500

Reply:
But we don't really want that, do we?

Reply:
It's a malformed request. Do you want to return some response with information?

Reply:
Indicating exactly what caused it (param name) would be good

Reply:
I'll mimic the alpha goerli response

Reply:
Fixed in e9e168cf2b327d9b1a80bf0380864fd6213e3724